module-bsp/board/rt1051/puretx/bsp/battery_charger/battery_charger.cpp

```diff
@@ -476,26 +476,8 @@ namespace bsp::battery_charger
             std::uint8_t temperatureInt = value.second >> 8;
             return utils::twosComplimentToInt(temperatureInt);
         }
-        /*
-        int getCurrentMeasurement()
-        {
-            auto value = fuelGaugeRead(Registers::Current_REG);
-            int current;
-            // 2's compliment into decimal
-            if (value.second & 0x8000) {
-                // negative numbers
-                std::bitset<16> currentBitset = std::bitset<16>(value.second - 1);
-                currentBitset.flip();
-                current =
-                    static_cast<int>((static_cast<std::uint16_t>(currentBitset.to_ulong()) * -1) * currentSenseGain);
-            }
-            else {
-                // positive numbers
-                current = static_cast<int>(value.second * currentSenseGain);
-            }
-            return current;
-        }
 
+        /*
```

Comment:
leftover?

Reply:
Some old leftover. It is not the point of this PR but I can clean it ;)

Reply:
done